src/modules/navigator/navigator_main.cpp

```diff
@@ -455,6 +458,16 @@ Navigator::run()
 
 				// CMD_NAV_TAKEOFF is acknowledged by commander
 
+			} else if (cmd.command == vehicle_command_s::VEHICLE_CMD_NAV_VTOL_TAKEOFF) {
+
+				_vtol_takeoff.setTransitionAltitudeAbsolute(cmd.param7);
+
+				// after the transition the vehicle will establish on a loiter at this position
+				_vtol_takeoff.setLoiterLocation(matrix::Vector2d(cmd.param5, cmd.param6));
+
+				// loiter height is the height above takeoff altitude at which the vehicle will establish on a loiter circle
+				_vtol_takeoff.setLoiterHeight(cmd.param1);
```

Comment:
That's not aligned with current mavlink specs right? Do you plan to open a mavlink PR for it, and meanwhile use VTO_LOITER_ALT instead?

Reply:
@sfuhrer Yes, exactly. The value is currently not used, we still stick to the parameter. In order to use the value from the command we need to extend the mavlink specs and also add some UI in QGC for specifying the altitude.